src/terminal_ui.cc

```diff
@@ -1263,6 +1267,7 @@ void TerminalUI::restore_terminal()
     fputs("\033[?1004l", stdout);
     fputs("\033[?25h", stdout);
     fputs("\033>", stdout);
+    // reset SGR
```

Comment:
Again, we might as well explain all the way:
```suggestion
    // set the terminal output back to default colours and style
```